app/_src/kubernetes-ingress-controller/guides/high-availability/fallback-config.md

```diff
@@ -0,0 +1,480 @@
+---
+title: Fallback Configuration
+type: explanation
+purpose: |
+  Explain how the Fallback Configuration feature works
+---
+
+In this guide you'll learn about the Fallback Configuration feature. We'll explain its 
+implementation details and provide an example scenario to demonstrate how it works in practice.
+
+{% include /md/kic/prerequisites.md release=page.release disable_gateway_api=false %}
+
+## Overview 
+
+{{site.kic_product_name}} in version 3.2.0 introduced the Fallback Configuration
+feature. It is designed to isolate issues related to individual parts
+of the configuration, allowing updates to the rest of it to proceed with no
+interruption. If you're using {{site.kic_product_name}} in a multi-team environment, the
+fallback configuration mechanism can help you avoid lock-ups when one team's
+configuration is broken.
+
+{:.note}
+> **Note:** The Fallback Configuration is an opt-in feature. You must
+> enable it by setting `FallbackConfiguration=true` in the controller's feature
```

Comment:
Can `enable it` be a link to the docs on how to do so?

Reply:
https://github.com/Kong/docs.konghq.com/pull/7463/commits/36b0389f32ab2fcc73257894afe6e40cb80c77c1